### ISSUE TITLE
Disable 'Save as PDF' option

### ISSUE THREAD

**Issue by ydotmalik** [DavBfr/dart_pdf#439]
In the print preview page, how can I disable 'Save as PDF' option? I do not want the user to be able a PDF copy.

**Comment by ydotmalik** [comment]
I see that this an platform level issue: #406 

Is it possible to encrypt/password/obscure the file such that it is printable but not readable through a viewer?

**Comment by DavBfr** [comment]
@ydotmalik yes it's possible to encrypt and give only print permission, but a user will still be able to view the pdf.

**Comment by ydotmalik** [comment]
I don't mind if the user is able to view the PDF if they save it, but I don't want them to print it. Is this possible?

**Comment by DavBfr** [comment]
Yes, it's possible. Look at: https://pub.nfet.net/pdf_crypto/

**Comment by ydotmalik** [comment]
Ok. It seems this library requires credentials. This library isn't free, correct?

**Comment by DavBfr** [comment]
Correct.

**Comment by ydotmalik** [comment]
I'm not sure pdf_crypto will do what I want.

I created a password protected PDF file outside of dart_pdf and passed it to ```Printing.layoutPdf()```. I get the following exception:
```
E/AndroidRuntime(13348): java.lang.RuntimeException: Cannot print a password protected PDF file
E/AndroidRuntime(13348): 	at android.print.PrintManager$PrintDocumentAdapterDelegate$MyHandler.handleMessage(PrintManager.java:1101)
...
```

This is expected, but if I created a password protected PDF file using pdf_crypto and try to print it, won't I get the same behavior?

What I want to do:
- generate a PDF using dart_pdf
- pass it to ```Printing.layoutPdf()```
- since it isn't possible to prevent the user from saving the PDF from the print preview screen, I want to at least prevent the user from printing the saved PDF again. AFAIK, I have no control over the file once it is saved as PDF from the preview screen.
